modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -711,7 +787,7 @@ private CompletableFuture<Boolean> prepareTableResourcesAndLoadToZoneReplica(Cre
      * @param zoneId Zone id.
      * @return Future, which will complete when the table processor loaded to the zone replica.
      */
-    private CompletableFuture<Void> preparePartitionResourcesAndLoadToZoneReplica(
+    private void preparePartitionResourcesAndLoadToZoneReplica(
```

Comment:
you've changed return type to void, but still have @ return section

Reply:
Fixed